### DISCUSSION TITLE
Tuya Fan Controller

### DISCUSSION THREAD

**Discussion by nckslater** [tuya/tuya-home-assistant#759] [Q&A]
Hi, 

I'm wondering if this device is compatible with the Tuya integration? 

[Tuya Fan Controller](https://www.efectoled.com/es/comprar-control-wifi/80078-controlador-wifi-para-ventilador-de-techo-dc.html)

Thanks

**Comment by cathelest** [comment]
Maybe, maybe not,
Without having one added to tuya to grab the control sets its hard to know,

Worst case you can probably add it to tuya app, create scenes and they will show up so will have some level of control, but may not get status updates etc, 


If its fsd category, support for tuya intergration is flaky,
I havd 3 fsd devices that dont work currently, but every device is different, and tuyas standards for devices are not at all standard with manufacturors changing control sets etc,
